ImageSharpCompare/ImageSharpCompare.cs

```diff
@@ -82,20 +82,45 @@ public static bool ImagesAreEqual(Stream actual, Stream expected)
         /// <param name="expected"></param>
         /// <returns>True if every pixel of actual is equal to expected</returns>
         public static bool ImagesAreEqual(Image actual, Image expected)
+        {
+            var ownsActual = false;
+            var ownsExpected = false;
+            Image<Rgb24>? actualPixelaccessableImage = null;
+            Image<Rgb24>? expectedPixelaccessableImage = null;
+            try
+            {
+                actualPixelaccessableImage = ToRgb24Image(actual, out ownsActual);
+                expectedPixelaccessableImage = ToRgb24Image(expected, out ownsExpected);
+
+                return ImagesAreEqual(actualPixelaccessableImage, expectedPixelaccessableImage);
+            }
+            finally
+            {
+                if (ownsActual)
+                    actualPixelaccessableImage?.Dispose();
```

Comment:
Do you have time to add tests that proof the expected behavior?

Reply:
Not at the moment, I will see if I can add later.